recipes/cocotb/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 959892eb94bd0b3ff40e0fca51d33a3936416deb853e2bac4f7f766b40002650 
+  patches:
+    - patches/0001-remove-static-stdc.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<310]
+  entry_points:
+    - cocotb-config = cocotb.config:main
+  run_exports:
+    - {{ pin_subpackage("cocotb", max_pin="x") }}
```

Comment:
This run_export tells me that your shared libraries are not python version specific because you haven't included a python version in the pinning.

Will your package still run if you move the shared libs from site-packages/cocotb/libs to $PREFIX/lib? This would allow you to build the shared libraries once per platform and use a `noarch: python` package for the python bits (which would reduce the build matrix to one build variant and allow users of your C API to avoid installing python).

Reply:
they are, but it wont. This package is a bit complicated due to the way it is used
https://github.com/conda-forge/staged-recipes/pull/21056#issuecomment-1309329721
https://github.com/cocotb/cocotb/pull/2268
https://github.com/ktbarrett/find_libpython